ui/components.d.ts

```diff
@@ -51,6 +55,8 @@ declare module 'vue' {
     NTree: typeof import('naive-ui')['NTree']
     NUpload: typeof import('naive-ui')['NUpload']
     NUploadDragger: typeof import('naive-ui')['NUploadDragger']
+    NUploadFileList: typeof import('naive-ui')['NUploadFileList']
+    NUploadTrigger: typeof import('naive-ui')['NUploadTrigger']
     NWatermark: typeof import('naive-ui')['NWatermark']
     RouterLink: typeof import('vue-router')['RouterLink']
     RouterView: typeof import('vue-router')['RouterView']
```

Comment:
The current codebase contains some outdated components like the "Kubernetes" namespace which is not relevant anymore. It's recommended to update all Vue.js dependencies from this year onwards to ensure compatibility with modern JavaScript features and best practices.

To optimize:

1. Remove unused or deprecated modules (replace them with more current ones if available).
2. Make sure there aren't too many unnecessary methods in each module that could be abstracted out for better reusability.
3. Ensure consistent formatting throughout the codebase using ESLint or prettier for clarity improvement and avoiding inconsistencies across files.